### PR TITLE
8053 refactor incorrect local test

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

Migrating  test from protractor to wdio.

medic/cht-core #7723 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8053-refactor-incorrect-local-test/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8053-refactor-incorrect-local-test/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8053-refactor-incorrect-local-test/docker-compose/cht-couchdb-clustered.yml)
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
@tatilepizs Just checking that you saw this?

**Review by tatilepizs — CHANGES_REQUESTED** [review]
This looks great @ngaruko !
Thank you for doing this. I just left a couple of question/suggestion.

File: tests/e2e/default/enketo/pregnancy.wdio-spec.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ describe('Pregnancy registration', () => {
 
     await commonPage.goToTasks();
     const tasks = await tasksPage.getTasks();    
-    expect(tasks.length).to.equal(3);
+    expect(tasks.length).to.equal(3);//TODO: investigate why 'Pregnancy home visit' task is not being generated
```
Comment (tatilepizs) [inline tests/e2e/default/enketo/pregnancy.wdio-spec.js:105]: I think that this comment can also be removed right? 🙂

**Review by tatilepizs — APPROVED** [review]
Looks good!
Thank you @ngaruko  for the changes 🎉 
I just leave a small suggestion

**Pull request merged**